### PR TITLE
Display collections

### PR DESCRIPTION
**Pull request by danlee1025**
- Added page to display collections
    - Are there specific headers we wanted for each collection?

I still need to add some styling and prop validation but the basic functionality for displaying the db collections is ready for review.

### CODE REVIEW

File: src/ui/components/Collections.jsx
Comment (blangley28) [inline src/ui/components/Collections.jsx:16]: Is there a way we can expose this on the backend. That way instead of updating multiple places simply modifying `src/storage/collections.js` would update everything

File: src/ui/components/Collections.jsx
Comment (blangley28) [inline src/ui/components/Collections.jsx:34]: FHIR resource collections should have `fullUrl` too since that is now their "key"

File: src/ui/components/Collections.jsx
Comment (blangley28) [inline src/ui/components/Collections.jsx:25]: Consider using react-query `useQuery` here. It won't really change anything now but it will allow us to easily update the display when the data changes.

File: src/ui/components/ResourceCell.jsx
Comment (blangley28) [inline src/ui/components/ResourceCell.jsx:14]: It would be really cool if you used something like https://www.npmjs.com/package/react-json-view here (this was just a quick google search but it seems easy enough to implement)

**Review by blangley28 — CHANGES_REQUESTED** [review]

File: src/ui/components/Collections.jsx
Comment (danlee1025) [reply src/ui/components/Collections.jsx:16]: yea I was thinking about this but we need to manually add in headers for each collection unless you can think of some generic headers for all the collections.

Comment (blangley28) [reply src/ui/components/Collections.jsx:16]: Hmm, yea without refactoring the database collections to be something like {fullUrl: 'blah', resource: {}} it would be hard... I honestly think we should do that but probably not here

File: src/ui/components/Collections.jsx
Comment (danlee1025) [reply src/ui/components/Collections.jsx]: Not sure, for now I added whatever is in `collections.js`... Won't be too hard to add if we decide we want to view them.

File: src/ui/components/Collections.jsx
Comment (danlee1025) [reply src/ui/components/Collections.jsx]: Updated to use this!

**Review by KeeyanGhoreshi — APPROVED** [review]

**Review by blangley28 — APPROVED** [review]
LGTM. Will merge once resource-fullurl has been approved.

**Pull request merged**
